syntax="proto3";

message LoginC2S {
    string userName = 1;
    string password = 2;
}